fgpyo/fasta/reference_set_builder.py

```diff
@@ -0,0 +1,174 @@
+"""
+Classes for generating Fasta files and records for testing
+----------------------------------------------------------------
+"""
+# import hashlib
```

Comment:
```suggestion
```

Reply:
Needed to create md5sums, is there another module you prefer?

Reply:
Ah, sorry, didn't see a TODO

Reply:
No worries!

Reply:
Removed in commit cb8ac832a05ba4a3ad304edb2de19c57d6f9b8f3